src/server/seeds/development/04_favorites.js

```diff
@@ -0,0 +1,37 @@
+exports.seed = function (knex) {
+  return knex('favorites')
+    .del()
+    .then(function () {
+      return knex('favorites').insert([
+        {
+          user_id: 1,
+          product_id: 1,
+        },
+        {
+          user_id: 1,
+          product_id: 5,
+        },
+        {
+          user_id: 3,
+          product_id: 8,
+          created_at: '2021-05-11 10:10:45',
+        },
+        {
+          user_id: 8,
+          product_id: 2,
+        },
+        {
+          user_id: 8,
+          product_id: 5,
+        },
+        {
+          user_id: 9,
+          product_id: 10,
+        },
+        {
+          user_id: 9,
+          product_id: 12,
+        },
+      ]);
```

Comment:
and all 3 above.

Reply:
Yes the created_at have not nullable property, but it have also the default timestamp value as now()